scripts/publish-cmds.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash -e
+
+if [[ -z "$1" ]]; then
+  echo -e "version is not set, run:\n"
+  echo "$0 [version] [prod|dev]"
+  exit 1
+fi
+
+if [[ -z "$2" ]]; then
+  echo -e "release type is not set, run:\n"
+  echo "$0 [version] [prod|dev]"
```

Comment:
wrong copy/paste here I guess.
I'm not so sure to understand the difference between the 2 args. Shouldn't the version be version of the docker image so latest/v0.20...

Reply:
the first argument is to specific the version (eg `v0.19`) and the second is to specify "where" to release, either dev or prod.

Reply:
```suggestion
  echo "$0 [release type] [prod|dev]"
```